### PR TITLE
Fix pytest

### LINKED ISSUE #24: Fix failing tests
Lots of tests are failing. For details see [here](https://github.com/DeepRank/deeprank-gnn-2/runs/5624296084?check_suite_focus=true)

### PR DESCRIPTION
**Pull request by svenvanderburg**
Fixes #24

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: [3.7, 3.8]
+        version: [3.8]
```
Comment (svenvanderburg) [inline .github/workflows/build.yml:13]: Any reason we need to support 3.7? It seems latest versions of major scientific packages like scipy, numpy, matplotlib do not support it any more.

**Comment by cbaakman** [comment]
The setup.py script is still failing. Will we handle this later?

**Comment by gcroci2** [comment]
> The setup.py script is still failing. Will we handle this later?

That's weird. Two commits ago (https://github.com/DeepRank/deeprank-gnn-2/pull/35/commits/d0c87dcc1fd3e747344c5a6336f6ff16abc37c51) all checks were passing, and I don't see significant changes in the last two commits.

**Comment by cbaakman** [comment]
> > The setup.py script is still failing. Will we handle this later?
> 
> That's weird. Two commits ago ([d0c87dc](https://github.com/DeepRank/deeprank-gnn-2/commit/d0c87dcc1fd3e747344c5a6336f6ff16abc37c51)) all checks were passing, and I don't see significant changes in the last two commits.

Did you run the setup.py script in a virtual environment?

**Comment by gcroci2** [comment]
> > > The setup.py script is still failing. Will we handle this later?
> > 
> > 
> > That's weird. Two commits ago ([d0c87dc](https://github.com/DeepRank/deeprank-gnn-2/commit/d0c87dcc1fd3e747344c5a6336f6ff16abc37c51)) all checks were passing, and I don't see significant changes in the last two commits.
> 
> Did you run the setup.py script in a virtual environment?

No, I meant that the checks were passing in the build done by GitHub Actions (see https://github.com/DeepRank/deeprank-gnn-2/runs/5661761424?check_suite_focus=true). That's why I am a bit confused

**Pull request merged**